polkadot/runtime/common/src/paras_registrar/mod.rs

```diff
@@ -578,7 +718,13 @@ impl<T: Config> Pallet<T> {
 
 		let deposit = deposit_override.unwrap_or_else(T::ParaDeposit::get);
 		<T as Config>::Currency::reserve(&who, deposit)?;
-		let info = ParaInfo { manager: who.clone(), deposit, locked: None };
+		let info = ParaInfo {
+			manager: who.clone(),
+			deposit,
+			locked: None,
+			billing_account: Some(who.clone()),
```

Comment:
Shouldn't this be `None`?

Reply:
Referring to the updated explanation in the code: 
```
/// The billing account of a parachain. This account is responsible for holding the required
/// deposit for this registered parachain.
///
/// This account will be responsible for covering all associated costs related to performing
/// parachain validation code upgrades.
///
/// When a parachain is newly registered, this will be set to the parachain manager. However,
/// at a later point, this can be updated by calling the
/// `set_parachain_billing_account_to_self` extrinsic.
///
/// None indicates that the billing account hasn't been set, which is the case for all legacy
/// parachains. Attempting to schedule a parachain upgrade will fail if the billing account is
/// set to None, except in the case where the parachain is not a lease-holding parachain.
```

Reply:
Also, I added an explanation about this in the PR description to make this part clear.

Reply:
Ok, I understand now what you are referring to. I will fix this.